site/docs/v1/tech/core-concepts/editions-features.adoc

```diff
@@ -0,0 +1,93 @@
+---
+layout: doc
+title: Editions and Features
+description: An explanation of editions and features
+---
+:page-liquid:
+
+:sectnumlevels: 0
+
+== Overview
+
+include::docs/v1/tech/shared/_licensing-premium-features-intro.adoc[]
+
+* <<Editions and Features>>
+* <<Feature List>>
+* <<Feature Documentation>>
+
+== Editions and Features
+
+There are four different editions. The documentation covers all of them.
+
+* `Community` has all features not otherwise labeled.
+* `Starter` has all premium features, but some may have some numeric limits.
+* `Essentials` has all advanced and premium features, but some may have some numeric limits.
+* `Enterprise` has all enterprise, advanced and premium features and no limits.
+
+Please review the link:/pricing[pricing page] for much more information on editions and features.
+
+== Feature List
```

Comment:
It may be unavoidable, but I think that we are adding a lot of FusionAuth DSL / semantics here...

Reply:
Yeah, it's kinda the point. So we have someplace for folks to familiarize themselves. I think it might be unavoidable, unfortunately.